linux/jdk/redhat/src/packageTest/java/packaging/DnfOperationsTest.java

```diff
@@ -53,27 +53,30 @@ void packageSuccessfullyInstalled(String distribution, String codename) throws E
 			result = runShell(container, "dnf update -y");
 			assertThat(result.getExitCode()).isEqualTo(0);
 
-			result = runShell(container, "dnf install -y " + containerRpm);
-			assertThat(result.getExitCode()).isEqualTo(0);
-
-			result = runShell(container, "rpm -qi " + System.getenv("PACKAGE"));
-			assertThat(result.getExitCode()).isEqualTo(0);
-			if (System.getenv("JDKGPG") != null) {
-				assertThat(result.getStdout())
-					.contains("Name        : " + System.getenv("PACKAGE"))
-					.contains("Group       : java")
-					.contains("License     : GPLv2 with exceptions")
-					.contains("Signature   : RSA/SHA256,")
-					.contains("Relocations : /usr/lib/jvm/" + System.getenv("PACKAGE"))
-					.contains("Packager    : Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org>");
-			} else {
-				assertThat(result.getStdout())
-					.contains("Name        : " + System.getenv("PACKAGE"))
-					.contains("Group       : java")
-					.contains("License     : GPLv2 with exceptions")
-					.contains("Signature   : (none)")
-					.contains("Relocations : /usr/lib/jvm/" + System.getenv("PACKAGE"))
-					.contains("Packager    : Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org>");
+			// below part: only test x86_64 rpm package in docker container
```

Comment:
comment doesn't match the code, wouldn't all architectures include aarch64 etc?

Reply:
so, when explicit set `ARCH` to `x86_64` or all or do not set ARCH at all , this does tests on these asserts
at the moment (before this PR) we only have "all" so it does build x86_64+ s390+ppc64le etcetc, but the test only done for the x86_64 one, the rest are not tested , which is the same as `System.getenv("testArch") == "all"`